src/views/SpaceCreate.vue

```diff
@@ -442,13 +480,9 @@ onMounted(async () => {
           class="block w-full"
           primary
           data-testid="create-proposal-publish-button"
-          @click="
-            !termsAccepted && space.terms
-              ? (modalTermsOpen = true)
-              : handleSubmit()
-          "
+          @click="handleSubmit"
         >
-          {{ $t('create.publish') }}
+          {{ isEditing ? 'Save changes' : $t('create.publish') }}
```

Comment:
can `'Save changes'` be moved to `locales`?

Reply:
Yes, I'm starting to use them only when it reduces logic. Using less locals will make the transition into SX UI easier as they don't use any.